composer.json

```diff
@@ -25,7 +25,11 @@
         "psr/container": "^1.0",
         "psr/link": "^1.0",
         "psr/log": "~1.0",
-        "symfony/contracts": "^1.1.7|^2",
```

Comment:
Only this line should be changed. This allows the ci to use the patched contracts on PRs.

Reply:
^1.1.8

Reply:
All right, I'll change it back to the metapackage. Just for the record: The build was green! 😎